src/app/components/articles/articles.component.tsx

```diff
@@ -1,33 +1,34 @@
 import React from 'react';
 import { useStoreon } from 'storeon/react';
-import { Article } from '../../modules/content/content.state';
-import {
-    ContentLabel,
-    ContentTile,
-} from '../contentItem/contentTile.component';
+import { useEffect } from 'react';
+import { LoadingIndicator } from '../loadingIndicator/loadingIndicator.component';
+import { ContentLabel } from '../../modules/content/content.state';
+import { CategoryItems } from '../categoryItems.component.tsx/categoryItems.component';
 import './articles.component.scss';
 
 export function Articles(): JSX.Element {
     const { content } = useStoreon('content');
 
-    function renderArticles(): JSX.Element {
-        if (!content || !content.articles.length) {
-            return <div>No data</div>;
-        }
+    useEffect(() => {}, [content.pending]);
 
-        return content.articles.map((article: Article, index: number) => (
-            <ContentTile
-                key={index + article.title}
-                contentItem={article}
-                contentLabel={ContentLabel.Article}
-            />
-        ));
+    if (content.pending) {
+        return <LoadingIndicator />;
     }
-
-    return (
-        <div className="articles-component">
-            <h2>Articles</h2>
-            {renderArticles()}
-        </div>
-    );
+    if (content.articles.length) {
+        return (
+            content && (
+                <div className="articles-content">
+                    <h2>Articles</h2>
+                    <div className="articles-items">
+                        <CategoryItems
+                            categoryItems={content.articles}
+                            label={ContentLabel.Article}
+                            specifiedLength={500}
+                        />
+                    </div>
+                </div>
+            )
+        );
+    }
+    return <div></div>;
```

Comment:
same

Reply:
Refactored, as mentioned in another comment.